src/main/java/lotto/domain/number/PayOut.java

```diff
@@ -18,17 +20,49 @@ public PayOut(String number) {
         this(new Number(number));
     }
 
+    private PayOut() {
+        value = new Number(0);
+    }
+
     private void validateNegative(int value) {
         if (value <= 0) {
             throw new IllegalArgumentException("입력값이 양수가 아닙니다.");
         }
     }
 
     public int getGameCount() {
-        return payOut.getValueAsInt() / GAME_PRICE;
+        return value.getValueAsInt() / GAME_PRICE;
     }
 
     public int getValueAsInt() {
-        return payOut.getValueAsInt();
+        return value.getValueAsInt();
+    }
+
+    public PayOut subtraction(PayOut payOut) {
+        Number subtractedNumber = value.subtraction(payOut.value);
+
+        if(subtractedNumber.equals(ZERO)) {
```

Comment:
Number와 PayOut에 대한 equals는 무슨비교일까요?
다지우고 메서드 전체를 이렇게 한줄로 써도 같지 않을까요?
```suggestion
        return new PayOut(value.subtraction(payOut.value));
```

Reply:
PayOut이 원래 0을 갖는것을 금지했는데, 허용하면서 리뷰어님 의견을 반영시켰습니다!